data/_models/shared/Processes.csv

```diff
@@ -1,51 +1,110 @@
-Sequencing runs,,observed insert size,int,,,,,,,,,https://w3id.org/fair-genomes/resource/FG_0000002,"In paired-end sequencing, the DNA between the adapter sequences is the insert. The length of this sequence is known as the insert size, not to be confused with the inner distance between reads. So, fragment length equals read adapter length (2x) plus insert size, and insert size equals read lenght (2x) plus inner distance.","Patient registry, FAIR Genomes"
-Sequencing runs,,percentage Q30,decimal,,,,,,,,,http://purl.obolibrary.org/obo/GENEPIO_0000089,"Percentage of reads with a Phred quality score over 30, which indicates less than a 1/1000 chance that the base was called incorrectly.","Patient registry, FAIR Genomes"
-Sequencing runs,,percentage Tr20,decimal,,,,,,,,,https://w3id.org/fair-genomes/resource/FG_0000003,Percentage of the target sequence on which 20 or more unique reads were successfully mapped.,"Patient registry, FAIR Genomes"
-Sequencing runs,,other quality metrics,text,,,,,,,,,http://edamontology.org/data_3914,"Other NGS quality control metrics, including but not limited to (i) sequencer metrics such as yield, error rate, density (K/mm2), cluster PF (%) and phas/prephas (%), (ii) alignment metrics such as QM insert size, GC content, QM duplicated reads (%), QM error rate, uniformity/evenness of coverage and maternal cell contamination, and (iii) variant call metrics such as number of SNVs/CNVs/SVs called, number of missense/nonsense variants, common variants (%), unique variants (%), gender match and trio inheritance check.","Patient registry, FAIR Genomes"
-Sequencing runs,,flow cell,string,,,,,,,,,http://purl.obolibrary.org/obo/OBI_0400043,"Aparatus in the fluidic subsystem where the sheath and sample meet. Can be one of several types; jet-in-air, quartz cuvette, or a hybrid of the two. The sample flows through the center of a fluid column of sheath fluid in the flow cell.","Patient registry, FAIR Genomes"
-Sequencing runs,,sequencing lane identifier,string,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C182081,The numerical identifier for the lane or machine unit where a sample was located during nucleotide sequencing.,"Patient registry, FAIR Genomes"
+Observations,,location of observation,ref,,,CatalogueOntologies,Organisations,,,,,http://purl.obolibrary.org/obo/NCIT_C119804,The site in or on the body at which the observation is made.,"Patient registry, FAIR Genomes"
+Experiments,Processes,,,,,,,,,,,http://semanticscience.org/resource/SIO_000994,"An experiment is an investigation that has the goal of verifying, falsifying, or establishing the validity of a hypothesis.","Patient registry, FAIR Genomes"
+Experiments,,General Sample Information,heading,,,,,,,,,,,"Patient registry, FAIR Genomes"
```

Comment:
I think you should reconsider headings a bit. Some suggestions on regrouping:
- general information = individuals, sample type, pathological state, sample id, local sample id, aliquot id, sample availability, volume (unit)
- 'sample collection' = age at collection, anatomical location (other), failed quality control, collected at organisation, collected by, date of collection

This doesn't block merge.